kujira/chain.json

```diff
@@ -33,19 +33,19 @@
   },
   "codebase": {
     "git_repo": "https://github.com/Team-Kujira/core",
-    "recommended_version": "v0.8.4-mainnet",
+    "recommended_version": "v0.8.7",
     "compatible_versions": [
-      "v0.8.4-mainnet"
+      "v0.8.7"
     ],
     "genesis": {
       "genesis_url": "https://github.com/Team-Kujira/networks/raw/master/mainnet/kaiyo-1.json"
     },
     "versions": [
       {
-        "name": "v0.8.4-mainnet",
-        "recommended_version": "v0.8.4-mainnet",
+        "name": "v0.8.7",
```

Comment:
this name shouldn't be changed, the `name` is the cosmovisor upgrade name

Reply:
thank you for clarifying that!